buildkite/build_project.sh

```diff
@@ -267,6 +269,23 @@ case "$REPO_FULL_NAME" in
         rm -rf "$TMP"
```

Comment:
Why not add dlang/phobos+no-autodecode to the list above and then distinguish at the end?
(everything else just looks duplicated)


```suggestion
        cd "$(basename "${REPO_FULL_NAME}")"&& make -f posix.mak clean 
        if [ "$REPO_FULL_NAME" == "dlang/phobos+no-autodecode" ] ; then
          make -f posix.mak -j2 autodecode-test
        else
          make -f posix.mak -j2 buildkite-test
        fi
        rm -rf "$TMP"
```

Reply:
I need to build phobos with `NO_AUTODECODE=1`. It's looking more and more like a mess if I have another `if` above.

Reply:
If this is what it takes to get it merged, I can do it. Just let me know.

Reply:
I have no strong preference/objection.